src/utils/isModelValueType.ts

```diff
@@ -1,14 +1,16 @@
+import isObjectLike from 'lodash/isObjectLike'
 import { ModelValueType, PrimitiveValueType } from '../glossary'
 
 function isPrimitiveValueType(value: any): value is PrimitiveValueType {
   return (
     typeof value === 'string' ||
     typeof value === 'number' ||
     typeof value === 'boolean' ||
+    isObjectLike(value) ||
```

Comment:
Can we use an already existing `isObject` utility?

https://github.com/mswjs/data/blob/5be9ca36f560bc2977ae68ea96651b4cd7ad0250/src/utils/isObject.ts

Reply:
Done - [704c03a](https://github.com/mswjs/data/pull/263/commits/704c03aec83a92026ba707cbb7350ea4f1afc7c2)